pom.xml

```diff
@@ -173,6 +173,24 @@
     </profiles>
 
     <build>
+      <filters>
```

Comment:
Did these changes help you fix your issue with profiles? If not, can we get rid of them?

Reply:
No, I just must moved the filters up so you could see it with the profiles.

Reply:
That makes sense